kits/cpp/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM ubuntu:20.04
+ARG DEBIAN_FRONTEND=noninteractive
```

Comment:
what does this do?

Reply:
During update/upgrade of Ubuntu, it will ask you for your timezone and expect input from stdin (thus hanging the docker build process). This argument instructs Ubuntu to run in noninteractive mode, meaning it will not prompt for any input.
Btw, what Ubuntu version is kaggle running on? I assumed it would be a recent version, so i used 20.04 as a placeholder.

Reply:
Should be 18.04, based on the image we provided last year for cpp kits